### PR TITLE
Add 500 page

### PR DESCRIPTION
**Pull request by pondohoo**
I'm new to web development so I'm not sure if I missed something necessary for the page

### CODE REVIEW

File: pages/500.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const error500: React.FC = () => {
+  return (
+    <div className="h-screen p-16 flex flex-col px-5 mx-auto justify-center items-center">
+      <h1 className="mb-10 font-extrabold text-8xl">500</h1>
+      <p className="font-semibold text-2xl">Uh oh. Something broke on our end. Check back later.</p>
+    </div>
+  );
+};
+
+export default error500
```
Comment (minsooerickim) [inline pages/500.tsx]: What you did works here too but just to keep everything more consistent within our project, we're going to stick with exporting with the function declaration. So remove line 12 and just replace your function declaration (line 3) with `export default function error500() { ... }`

File: pages/500.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const error500: React.FC = () => {
+  return (
+    <div className="h-screen p-16 flex flex-col px-5 mx-auto justify-center items-center">
+      <h1 className="mb-10 font-extrabold text-8xl">500</h1>
+      <p className="font-semibold text-2xl">Uh oh. Something broke on our end. Check back later.</p>
```
Comment (minsooerickim) [inline pages/500.tsx]: we don't need any styling for `<p>` `<h1>` tags here. We're going to define a global styling that will be automatically applied to all `<p>` and` <h1>` tags

File: pages/500.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const error500: React.FC = () => {
+  return (
+    <div className="h-screen p-16 flex flex-col px-5 mx-auto justify-center items-center">
```
Comment (minsooerickim) [inline pages/500.tsx]: generally, we would use `<section>` tag for this scenario and when you create headers, footers, etc. You would use `<div>` tag when you "divide" something in the document. It pretty does the same thing in this case, but I would just change it to use `<section>`

**Comment by minsooerickim** [comment]
@pondohoo great work btw!!

**Comment by minsooerickim** [comment]
also fyi, once you address the comments, just make a reply and @ me

**Comment by pondohoo** [comment]
@minsooerickim I think all of the issues you mentioned, could you double check?

**Comment by minsooerickim** [comment]
> @minsooerickim I think all of the issues you mentioned, could you double check?

looks good to me

**Comment by minsooerickim** [comment]
one more thing, can you use the `<Page>` component under pages as a wrapper. Lmk if you have any questions about this

**Comment by pondohoo** [comment]
@minsooerickim just added the page wrapper, how does it look now?

**Comment by minsooerickim** [comment]
@pondohoo you should set your linter as prettier; Prettier is a linter (auto formats your code in a consistent style we are using). Oncee you set it, it should reformat your code for you. I'm going to go over setting that up in our onboarding meeting next week but if you want you can reference this post to set up prettier https://stackoverflow.com/a/64273353

File: pages/500.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Page } from '@/components/Page/Page'
+
+export default function error500()
+{
+  return (
+    <Page>
+      <section className="h-screen p-16 flex flex-col px-5 mx-auto justify-center items-center">
```
Comment (minsooerickim) [inline pages/500.tsx]: if i make the screen size smaller, the texts aren't centered and the x axis margin is a bit off in some screen sizes

**Comment by pondohoo** [comment]
@minsooerickim I think I configured Prettier now, let me know if my formatting is okay.

It looks like it aligns when I shrink the screen now, and I think I fixed the x-axis issue?

File: pages/500.tsx (hunk 4 above)
Comment (minsooerickim) [reply pages/500.tsx]: i still see this
Screen Shot 2022-07-24 at 2 01 14 PM

Comment (minsooerickim) [reply pages/500.tsx]: I think you need `text-center`

**Comment by pondohoo** [comment]
@minsooerickim I had added text-center 3 days ago but I'm not sure why my second commit didn't get pushed, my bad. Does it look right now?

**Comment by minsooerickim** [comment]
@pondohoo everything looks good! I just fixed the padding a bit and prettier stuff. I'll go over prettier stuff this Wednesday during the onboarding meeting. Great work!! 🥳

**Pull request merged**